xbmc/platform/linux/test/TestSysfsPath.cpp

```diff
@@ -8,24 +8,38 @@
 
 #include "platform/linux/SysfsPath.h"
 
+#include <chrono>
+#include <string>
+#include <sstream>
 #include <fstream>
 
 #include <gtest/gtest.h>
 
 struct TestSysfsPath : public ::testing::Test
 {
-  ~TestSysfsPath() { std::remove("/tmp/kodi-test"); }
+  std::string GetTestFilePath()
+  {
+    using namespace std::chrono;
 
-  std::ofstream m_output{"/tmp/kodi-test"};
+    std::stringstream ss;
+    unsigned long long timestamp = duration_cast< nanoseconds >(
+      system_clock::now().time_since_epoch()).count();
+
+    ss << "/tmp/kodi-test" << timestamp;
```

Comment:
Resulting path might be more readable if you add a dash between `test` and the timestamp:

```
"/tmp/kodi-test-" << timestamp
```

Reply:
Done! As well as Wolfgang's point.